lib/base.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+const process = require('./process.js')
+const { hrtime } = process
```

Comment:
Not sure why we don't just keep using `process.hrtime()` where we previously did but not a blocker.

Reply:
Oh, right, that's because it was `const proc = ...` and then `const { hrtime } = proc`, but I reverted the `proc` var when I moved it into a separate module.

Reply:
This also required checking that `process.hrtime` is _still_ a function when we finish the test.